test/3_records_test.js

```diff
@@ -497,14 +497,15 @@ contract('LinniaRecords', accounts => {
       const tx0 = await instance.addRecord(testDataHash, testMetadata, testDataUri, {
         from: user
       });
-      assert.equal(tx0.receipt.status, '0x01');
+      // sometimes retuns 0x01 -- not sure why
+      assert.equal(parseInt(tx0.receipt.status, 16), 1);
```

Comment:
This was is better but remove the comment on top ^

Reply:
+1